### PR TITLE
10/21 Email

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2022-10-21.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+
+### How can you support us?
+
+More an more, people love what we're doing and ask how we make money. The dirty little secret is, as with most start ups, we don't make much money. Looking at how similar big companies make most of their money, there are two potential businesses for us in the long run, enterprise DoltHub/DoltLab or Hosted Dolt. Maybe we'll invent a new big business in the database space but that's unlikely. 
+
+In the short run, we have a bunch of ways you can support us. Here's a list from cheapest on up.
+
+1. **Free** - Use the product. Run a Dolt database and share it with your teammates on DoltHub or DoltLab.
```
Comment (bpf120) [inline emails/2022-10-21.md:35]: "use the product and tell us about it. we love hearing how creative you are all."

File: emails/2022-10-21.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+2. **Free** - Cut [GitHub Issues](https://github.com/dolthub/dolt/issues) for bugs you find or improvements you want.
```
Comment (bpf120) [inline emails/2022-10-21.md:36]: Add a note about coming to Discord for support from the team.

File: emails/2022-10-21.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+3. **Free to start** - Sign up fir DoltHub Pro. You'll pay $50/month after you have over 1GB of databases on DoltHub.
+4. **Any Amount** - [GitHub sponsor the DoltHub organization](https://github.com/sponsors/dolthub).
+5. **$50/month and up** - Deploy a [Hosted Dolt](https://hosted.doltdb.com) Instance. Build a production application on it.
+6. **$5,000/month** - Buy support for Dolt, DoltHub, DoltLab and all our other products. This price is negotiable depending on your support needs.
```
Comment (bpf120) [inline emails/2022-10-21.md:40]: short blurb on what you get with support maybe

File: emails/2022-10-21.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+**E**vent-driven serverless platforms
+
+Honestly, this is some weak acronym play compared to the classics [JAM](https://en.wikipedia.org/wiki/Jamstack), [MERN](https://www.mongodb.com/mern-stack), or the OG, [LAMP](https://en.wikipedia.org/wiki/LAMP_(software_bundle)). But it got us thinking what would be a good Dolt stack. After some clever word play, the people who brought you Dolt give to you the DORK stack!
+
+**D**olt
+**O**RM
+**R**eact
+**K**ubernetes
+
+This stack powers the [Hosted Dolt])(https://hosted.doltdb.com) website. Have a better one? Reply to this email and I'll shout it out next week.
```
Comment (tbantle22) [inline emails/2022-10-21.md:17]:
```suggestion
This stack powers the [Hosted Dolt](https://hosted.doltdb.com) website. Have a better one? Reply to this email and I'll shout it out next week.
```

File: emails/2022-10-21.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### Better Alias Support
+
+SQL aliases have been a thorn in Dolt's side since launch. They mostly worked but often would fail in strange ways. A query wouldn't work unless you added an alias. It was frustrating. 
+
+[Jason](https://www.dolthub.com/team#jason) has spent the past month or so [upping our alias game](https://www.dolthub.com/blog/2022-10-21-alias-improvements/). It's better now. There are still bugs though. If you find one, [cut an issue](https://github.com/dolthub/dolt/issues). We'll fix it with high priority.
+
+### How can you support us?
+
+More an more, people love what we're doing and ask how we make money. The dirty little secret is, as with most start ups, we don't make much money. Looking at how similar big companies make most of their money, there are two potential businesses for us in the long run, enterprise DoltHub/DoltLab or Hosted Dolt. Maybe we'll invent a new big business in the database space but that's unlikely.
```
Comment (tbantle22) [inline emails/2022-10-21.md:33]:
```suggestion
More and more, people love what we're doing and ask how we make money. The dirty little secret is, as with most start ups, we don't make much money. Looking at how similar big companies make most of their money, there are two potential businesses for us in the long run, enterprise DoltHub/DoltLab or Hosted Dolt. Maybe we'll invent a new big business in the database space but that's unlikely. 
```

**Review by tbantle22 — APPROVED** [review]

**Pull request merged**